modules/adding-tab-pods-page.adoc

```diff
@@ -46,10 +47,15 @@ Custom plugin code is not supported by Red Hat. Only link:https://access.redhat.
   }
 }
 ----
-<1> Update the name of your plugin.
-<2> Update the version.
-<3> Update the display name for your plugin.
-<4> Update the description with a synopsis about your plugin.
++
+--
+where:
+
+`"name": "my-plugin",`:: Update the name of your plugin.
+`"version": "0.0.1",`:: Update the version.
+`"displayName": "My Plugin",`:: Update the display name for your plugin.
+`"description": "Enjoy this shiny, new console plugin!",`:: Update the description with a synopsis about your plugin.
```

Comment:
I believe we are supposed to use the full parameter [according to the SSG](https://redhat-documentation.github.io/supplementary-style-guide/#explain-commands-variables-in-code-blocks). See"bulleted list explaining the structure of an YAML file"
```suggestion
where:

`consolePlugin.name.my-plugin`:: Update the name of your plugin.
`consolePlugin.version.0.0.1`:: Update the version.
`consolePlugin.displayName.My Plugin`:: Update the display name for your plugin.
`consolePlugin.description.Enjoy this shiny, new console plugin!`:: Update the description with a synopsis about your plugin.
```